docs/users/cli-usage.md

```diff
@@ -228,9 +228,9 @@ ddev exec touch public/FIRST_INSTALL
 ddev launch
 ```
 
-**On TYPO3 versions < 9 an install may fail if you use the https URL to install because the allowed proxy is not configured yet ("Trusted hosts pattern mismatch"). Please use "http" instead of "https" for the URL while doing the install.**
+**Note**
 
-To connect to the database within the database container directly, please see the [developer tools page](developer-tools.md#using-development-tools-on-the-host-machine).
+On TYPO3 versions < 9 an install may fail if you use the https URL to install because the allowed proxy is not configured yet ("Trusted hosts pattern mismatch"). Please use "http" instead of "https" for the URL while doing the install.
```

Comment:
Do you think we can just remove this part now anyway?

Reply:
I wanted to say "TYPO3 v9 is still supported commercially in the TYPO3 ELTS programme."

But then I just tested with a v9 and there is no error anymore with https:// :)

I'll remove it.